.github/workflows/continuous-integration-dotnet.yml

```diff
@@ -0,0 +1,84 @@
+name: Continuous Integration - .NET
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - services/**
+  pull_request:
+    paths:
+      - services/**
+
+jobs:
+  pre_ci:
+    name: Prepare CI environment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: '[Push] Get commit message'
+        if: github.event_name == 'push'
+        id: push_get_commit_message
+        run: echo ::set-output name=push_commit_message::$(git log --format=%B -n 1 HEAD)
+      - name: '[Pull Request] Get commit message'
+        if: github.event_name == 'pull_request'
+        id: pr_get_commit_message
+        run: echo ::set-output name=pr_commit_message::$(git log --format=%B -n 1 HEAD^2)
+      - name: Add problem matchers
+        run: |
+          echo "::add-matcher::.github/problemMatchers/tsc.json"
```

Comment:
we have tsc problem matchers for dotnet - this might cause issues.

Reply:
infact - @actions/setup-dotnet registers it's own problem matchers, so we don't even need to use external ones.